test/utf8/rlexe.xml

```diff
@@ -20,4 +20,17 @@
       <baseline />
```

Comment:
we need to remove `<baseline />` in order to make result from this test case effective